src/main/java/com/exadel/sandbox/entities/Employee.java

```diff
@@ -0,0 +1,35 @@
+package com.exadel.sandbox.entities;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.FieldDefaults;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@Entity
+public class Employee {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    Integer emId;
+
+    @OneToOne
+    @JoinColumn(name = "tg_info_id")
+    TgInfo tgInfo;
+
+    String emFirstname;
```

Comment:
What about @Column? does emFurstName saved in DB? I think now only id will be saved in db. Please check other pleaces too

Reply:
it was working because the name was equal. in DB it was em_firstname I think that's why it worked. now I'm fixing these.